src/app/cli/src/init/graphql_queries.ml

```diff
@@ -154,11 +154,13 @@ module Send_payment =
 mutation ($sender: PublicKey!,
           $receiver: PublicKey!,
           $amount: UInt64!,
-          $token: UInt64,                                                                                                                                                                                                                              $fee: UInt64!,
+          $token: UInt64,
+          $fee: UInt64!,
           $nonce: UInt32,
           $memo: String) {
   sendPayment(input:
-    {from: $sender, to: $receiver, amount: $amount, token: $token, fee: $fee, nonce: $nonce, memo: $memo}) {
+    {from: $sender, to: $receiver, amount: $amount, token: $token, fee: $fee, nonce: $nonce, memo: $memo},
+    ignoreWarnings: true) {
```

Comment:
Can you accept this as an argument and surface it at the command line level, so that an `--ignore-warnings true` flag has to be passed there too?

Reply:
Yep, wasn't sure if I should squeeze that in here or do it in a follow-up PR but it should be quick (and it's maybe the easiest way for me to test the other changes work anyway)